transport_and_format.asciidoc

```diff
@@ -312,81 +312,83 @@ include::examples/transport_and_format_get_token_list_example.json[]
 
 [cols="4,14",options="header"]
 |===
 |HTTP Header |Description
 
-|OCPI-to-party-id |'party id' of the connected party this messages is to be send to.
-|OCPI-to-country-code |'country code' of the connected party this messages is to be send to.
-|OCPI-from-party-id |'party id' of the connected party this messages is send from.
-|OCPI-from-country-code |'country code' of the connected party this messages is send from.
+|OCPI-to-party-id |'party id' of the connected party this message is to be sent to.
+|OCPI-to-country-code |'country code' of the connected party this message is to be sent to.
+|OCPI-from-party-id |'party id' of the connected party this message is sent from.
+|OCPI-from-country-code |'country code' of the connected party this message is sent from.
 |===
 
 .Example sequence diagram of a GET for 1 Object from a CPO to an eMSP.
 image::images/sd_get_simple.svg[OCPI Sequence Diagram Hub GET]
 
 .Example sequence diagram of a PUT via 2 Hubs.
-image::images/sd_put_2_hubs.svg[OCPI Sequence Diagram Hub PUT with 2 hubs]
+image::images/sd_put_2_hubs.svg[OCPI Sequence Diagram Hub PUT with 2 Hubs]
 
 
 ===== Omitting from address in responses
 
-When a party responses to a request (via a hub), and it was a request that contained the 'OCPI-to-' headers,
-the 'OCPI-from-' headers might be omitted. The requesting party knows to which party the request was send.
-It might be good practice to always provide the 'OCPI-from-' headers in a response,
-just to be complete and to give a requesting party to change to validate if the request was routed correctly by the Hub.
+When a party responds to a request (via a Hub), and it has been a request that contained the 'OCPI-to-' header,
+the 'OCPI-from-' header MAY be omitted. The requesting party knows to which party the request was sent.
+It might be good practice to always provide the 'OCPI-from-' header in a response though,
+just to be complete and to give the requesting party a chance to validate if the request has been routed correctly by the Hub.
 
 
 [[transport_and_format_message_routing_broadcast_push]]
 ===== Broadcast push
 
 For simplicity, connected clients might PUSH (POST, PUT, PATCH) information to all connected clients with an "opposite role",
 CPO pushing information to all eMSPs and NSPs. eMSP pushing information to all CPOs.
-(the role "Other" is seen as an eMSP type of role, so broadcast push from a CPO is also send to Other,
-messages from "Other" are send to CPOs, not the eMSPs.)
+(The role "Other" is seen as an eMSP type of role, so broadcast push from a CPO is also sent to "Other".
+Messages from "Other" are only sent to CPOs and not to eMSPs though.)
 
-Broadcast push might be very useful to push information like Locations or Tokens to all parties,
+Broadcast pushing data through a Hub might be very useful to share information like Locations or Tokens with all parties
```

Comment:
I would like to use 'Broadcast Push' as a term for a functionality. Maybe naming it Broadcast Push here would have made it more clear.

Reply:
Yes, it makes it more clear. Personally, I think the term _broadcast_ already implies that data is _pushed_ and not _pulled_, but if you want to keep the term, that's fine for me. I'll change the occurences accordingly.